src/main/java/io/github/sefiraat/slimetinker/items/tinkermaterials/setup/Traits.java

```diff
@@ -495,7 +495,7 @@ private Traits() {
         .setAddedBy(SupportedPluginsManager.CORE_NOTE)
         .setTraitName("Farmer")
         .setLore(
-            "All crop drops 1 extra item. Not effected by fortune.",
+            "All crop drops 1 extra item. Not affected by fortune.",
```

Comment:
```suggestion
            "All crops drop 1 extra item. Not affected by fortune.",
```

might as well fix my other mistake here eh ;).